source/documentation/api_key/api_key.md

```diff
@@ -0,0 +1,9 @@
+## Authentication 
+
+If you have created an API key then you can use it with all registers.
+
+Your API key should be of the form `76415efc-3d0f-4522-affd-bb6464e5ea88`. You should place it in requests as part of the `Authorization` header. For example, this is a valid request:
+
+```
+curl https://country.register.gov.uk/record/GB.json -H "Authorization: 76415efc-3d0f-4522-affd-bb6464e5ea88"
```

Comment:
We use `--header` rather than `-H` in all other examples so just wondering whether we should be consistent. I don't mind which one we pick.